.openpublishing.redirection.json

```diff
@@ -2404,6 +2404,10 @@
             "source_path": "docs/framework/windows-workflow-foundation/samples/property-grid-extensibliity.md",
             "redirect_url": "/dotnet/framework/windows-workflow-foundation/samples/property-grid-extensibility",
             "redirect_document_id": true
+        },
+        {
+            "source_path": "docs/csharp/language-reference/language-specification/index.md",
```

Comment:
nit-pick: the entry is not in alphabetical order by the `source_path`
recently @mairaw has ordered this file, but I see that entry above is also not in "its" spot.

Reply:
I also moved the workflow entry to the correct location.